README.md

```diff
@@ -25,6 +25,13 @@ $ composer global require hirak/prestissimo
 ```
 
 
+## Update
+
+```bash
+$ composer global update hirak/prestissimo --with-dependencie
```

Comment:
This would not work.

Reply:
It worked for me to update from ``0.3.9`` to ``0.3.10``. It might not work upgrading from ``0.3.x`` to ``0.4.x`` depending on how the version constraint is set.

Anyway, the point of this PR was to add documentation on how to receive new versions of the plugin. Something most Composer users aren't doing on a daily basis.

Reply:
I bet it was not. Simply copy this comment and try to run it in the command line to see the error.

Reply:
Sorry, my bad. I failed hard at copy & pasting 😆 
It missed a "s" at the end:

``composer global update hirak/prestissimo --with-dependencies``